titles/discover/master.adoc

```diff
@@ -5,12 +5,12 @@ include::artifacts/attributes.adoc[]
 :imagesdir: images
 :title: About {product}
 :subtitle: Introduction to {product}
-:abstract: {product} is a developer platform designed to build developer portals. Use {product} to provide a streamlined development environment with a centralized software catalog to build high-quality software efficiently. 
+:abstract: {product} is a developer platform designed to build developer portals. Use {product} to provide a streamlined development environment with a centralized software catalog to build high-quality software efficiently.
 
 //[id="{context}"]
 //= {title}
 
 //{abstract}
 
 
-//include::assemblies/assembly-about-rhdh.adoc[]
+include::assemblies/assembly-about-rhdh.adoc[]
```

Comment:
@jmagak 

Please see preview which looks like we are duplicating the _About Red Hat Developer Hub_ heading, to avoid this I'd suggest maybe adding a leveloffset to the assembly include statement:

```suggestion
include::assemblies/assembly-about-rhdh.adoc[leveloffset=+1]
```
Without the offset I think it will render as something like _Part I. About Red Hat Developer Hub_ which I'm not sure looks great with minimal content like we have currently in this title.